src/__test__/components/MetricDetails/MetricDetails.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render  } from '@testing-library/react';
+
+import { mockData, district, metric, metricBar } from './mockData';
+import MetricDetails from '@components/MetricDetails';
+
+jest.mock('../../../data/Data.json', () => (mockData));
+
+
+describe('MetricDetails component', () => {
+  it('should be able to render the MetricDetails component without bar', () => {
+    const { getByText } = render(
```

Comment:
**pattern:** please use the `screen` object from testing-library instead to get the verification function from the render destructor